File: Project3/task3.rb
class Rectangle
	def initialize(width,height)
		@width=width
		@height=height
	end
	def area()
		return @width*@height
	end
	def perimeter()
		return 2*@width+2*@height
	end
	def modfunc()
		return @width%(@height)
	end

recta=Rectangle.new(3.4,5.6)
rectb=Rectangle.new(6.8,9.7)

puts "recta area: #{recta.area()}"
 
 puts "rectb area: #{rectb.area()}"
 
 puts "recta perimeter: #{recta.perimeter()}"
 
 puts "rectb perimeter: #{rectb.perimeter()}"
 
 puts "recta widthmodheight: #{recta.modfunc()}"
 
 puts "rectb widthmodheight: #{rectb.modfunc()}"
 
 puts "Difference in perimeter rectb and rect a: #{rectb.perimeter()-recta.perimeter()}"
 
 puts "Ratio of areas of rectb and recta: #{rectb.area()/recta.area()}"
end

File: Project3/task3.cpp
// member initialization
#include <iostream>
using namespace std;

class Rectangle {
	int width,height;
public:
	Rectangle(double,double);
	double area() {return (width*height);}
	double perimeter() {return (2*width+2*height);}
	double modfunc() {return (width%height);}

};

Rectangle::Rectangle(double a,double b){
	width=a;
	height=b;
}

int main () {
  Rectangle recta (6.4,6.5);
  Rectangle rectb (7.9,8.3);
  cout << "recta area: " << recta.area() << endl;
  cout << "rectb area: " << rectb.area() << endl;
  cout << "recta perimeter: " << recta.perimeter() << endl;
  cout << "rectb perimeter: " << rectb.perimeter() << endl;
  cout << "recta widthmodheight: " << recta.modfunc() << endl;
  cout << "rectb widthmodheight: " << rectb.modfunc() << endl;
  cout << "Difference in perimeter rectb and rect a: " << rectb.perimeter()-recta.perimeter() <<endl;
  cout << "Ratio of areas of rectb and recta: " <<rectb.area()/recta.area() << endl;
  return 0;
}